SEPRTR/iocBoot/iocSEPRTR-IOC-01/st.cmd

```diff
@@ -0,0 +1,53 @@
+#!../../bin/windows-x64/SEPRTR-IOC-01
+
+## You may have to change SEPRTR-IOC-01 to something else
+## everywhere it appears in this file
+
+# Increase this if you get <<TRUNCATED>> or discarded messages warnings in your errlog output
+errlogInit2(65536, 256)
+
+< envPaths
+
+epicsEnvSet "STREAM_PROTOCOL_PATH" "$(SEPRTR)/data"
+epicsEnvSet "DEVICE" "L0"
+
+cd "${TOP}"
+
+## Register all support components
+dbLoadDatabase "dbd/SEPRTR-IOC-01.dbd"
+SEPRTR_IOC_01_registerRecordDeviceDriver pdbbase
+
+##ISIS## Run IOC initialisation 
+< $(IOCSTARTUP)/init.cmd
+
+## Device simulation mode IP configuration
+$(IFDEVSIM) drvAsynIPPortConfigure("$(DEVICE)", "localhost:$(EMULATOR_PORT=57677)")
+
+## For recsim:
+$(IFRECSIM) drvAsynSerialPortConfigure("$(DEVICE)", "$(PORT=NUL)", 0, 1, 0, 0)
+
+## Load record instances
+
+##ISIS## Load common DB records 
+< $(IOCSTARTUP)/dbload.cmd
+
+# Load up DB record for talking to the DAQ box
+epicsEnvSet("DAQMX","$(MYPVPREFIX)$(IOCNAME):DAQ")
```

Comment:
IBEX PV names typically start with the `$(P)` macro, which is often superceeded here with `$(DAQMX)`. If this is a necessary replacement, could the reason be clearly labelled somewhere?